packages/mercury/src/components/paginator/_paginator-styles.scss

```
@@ -0,0 +1,271 @@
+@use "../../../../common/resets" as *;
+@use "../../base/common" as *;
+@use "../../base/label/label-styles" as *;
+@use "../combo-box/combo-box-styles" as *;
+@use "../input/input-styles" as *;
+
+%paginator {
+  @extend %body-regular-s;
+  display: grid;
+  gap: var(--mer-spacing--xs);
+  grid-template: "items-per-page items-per-page-info . first__button prev__button go-to next__button last__button" 1fr / auto auto 1fr auto auto auto auto;
+
+  &__combo-box {
+    border: 1px solid var(---mer-border-color__primary--focused, #d2ddeb);
+    width: 100px;
+  }
+
+  &__go-to {
+    align-items: center;
+    display: inline-flex;
+    gap: var(--mer-spacing--sm);
+
+    &__input {
+      border: 1px solid var(---mer-border-color__primary--focused, #d2ddeb);
+    }
+  }
+
+  &__items-per-page {
+    align-items: center;
+    gap: var(--mer-spacing--md);
+
+    &__info {
+      @extend %body-regular-s;
+      color: var(--mer-text__complementary);
+      margin-inline-start: var(--mer-spacing--xs);
+    }
+  }
+
+  &__button {
+    align-items: center;
+    color: var(--mer-icon__neutral);
+    display: flex;
+    height: var(--mer-spacing--lg);
```

```suggestion
    block-size: var(--mer-spacing--lg);
```

Use logical properties instead of physical properties.